website/www/site/layouts/partials/head_homepage.html

```diff
@@ -0,0 +1,83 @@
+{{/*
```

Comment:
See comment above. I think this will now be the default template for every page type. Is that what we want?

Reply:
No, you totally right. Came with another approach.